src/Components/Pickers/Internal/Picker.base.styles.ts

```diff
@@ -8,22 +8,34 @@ const classNames = {
 export const getStyles = (_props: IPickerBaseStyleProps): IPickerBaseStyles => {
     return {
         root: [classNames.root],
-        button: [
-            classNames.button,
-            {
-                border: '1px solid var(--cb-color-input-border)',
-                borderRadius: '50%',
-                cursor: 'pointer',
-                height: 28,
-                width: 28
-            }
-        ],
+        button: [],
         subComponentStyles: {
             callout: {
                 calloutMain: {
                     // this is not exposed on the styles directly
                     '& .ms-swatchColorPickerBodyContainer': {
-                        minWidth: 'unset'
+                        minWidth: 'unset',
```

Comment:
did you yank this from teh fluent repo or how did you come up with all these styles?

Reply:
I looked at Fluent repo but the majority came from opening dev tools and looking at the DOM generated from the ColorSwatchPicker